hudi-cli/src/main/java/org/apache/hudi/cli/commands/ExportCommand.java

```diff
@@ -190,7 +191,7 @@ private int copyNonArchivedInstants(List<HoodieInstant> instants, int limit, Str
     final HoodieTableMetaClient metaClient = HoodieCLI.getTableMetaClient();
     final HoodieActiveTimeline timeline = metaClient.getActiveTimeline();
     for (HoodieInstant instant : instants) {
-      String localPath = localFolder + Path.SEPARATOR + instant.getFileName();
+      String localPath = localFolder + HoodieLocation.SEPARATOR + instant.getFileName();
```

Comment:
Maybe we should rename `HoodieLocation` to `HoodiePath` ? We already have `HoodieRecordLocation` which is a totally different notion, these two class would confuse people a lot.

Reply:
The [federated storage layer](https://github.com/apache/hudi/blob/master/rfc/rfc-60/rfc-60.md) wants to use this `HoodieLocation` for redirecting data files by adding a prefix so there's the translation of logical and physical storage location.  `HoodieLocation` is still different from `HoodieRecordLocation` as it represents file location.

Reply:
Okay, if we already reached concensus.